### PR TITLE
Add content partials for error-message, hint and help us improve

### PR DESCRIPTION
**Pull request by ethanWallace**
# Summary | Résumé

Add partials for repeated content.

- Error-message and hint for form documentation pages.
- helpus for the Help us improve section at the bottom pages

Note: Error-message and hint will still need French translations.

Also updated `gcds-components` to latest published version.

### CODE REVIEW

File: src/_data/errormessage.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      3: "Avoid placing hint text or error messages below the container where they might be hidden.",
+    }
+  },
+  fr: {
+    heading: "Keep hint text static and perceivable",
+    listitems: {
+      0: "Use the required attribute for fields that must be filled in. This will place “required” at the end of the field label. Avoid adding red text and additional symbols or punctuation, like the asterisk (*).",
+      1: "Use the error message component to address missing and invalid entries for all required inputs. Avoid using them for optional ones.",
+      2: "Place an inline error message just before the input (either to its immediate left or above it) so it's noticeable.",
+      3: "Avoid placing hint text or error messages below the container where they might be hidden.",
```
Comment (amazingphilippe) [inline src/_data/errormessage.js:17]: Could this be an array?

File: src/_data/hint.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  en : {
+    heading: "Keep hint text static and perceivable",
+    listitems: {
+      0: "Add hint text to help a person provide a complete value in the input and avoid an error state.",
+      1: "Use the hint text attribute to add the hint text below the label and above the field.",
+      2: "Avoid placing hint text in the container where it will disappear once the field is selected.",
```
Comment (amazingphilippe) [inline src/_data/hint.js:7]: Same!

File: src/_includes/partials/error-message.njk [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  </li>
+  <li>
+    {{ errormessage[locale].listitems.1 }}
+  </li>
+  <li>
+    {{ errormessage[locale].listitems.2 }}
+  </li>
+  <li>
+    {{ errormessage[locale].listitems.3 }}
+  </li>
```
Comment (amazingphilippe) [inline src/_includes/partials/error-message.njk:15]: Maybe we could loop? Would be nice if we decide to add items in the future.

File: src/_includes/partials/hint.njk [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+<ul class="list-disc">
+  <li>
+    {{ hint[locale].listitems.0 }}
+  </li>
+  <li>
+    {{ hint[locale].listitems.1 }}
+  </li>
+  <li>
+    {{ hint[locale].listitems.2 }}
+  </li>
```
Comment (amazingphilippe) [inline src/_includes/partials/hint.njk:12]: Same comment about looping.

**Review by amazingphilippe — APPROVED** [review]
Some suggestions, otherwise looks good to me!

File: src/_data/hint.js (hunk 2 above)
Comment (ethanWallace) [reply src/_data/hint.js:7]: Good idea! Will do that now

File: src/_includes/partials/helpus.njk [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+			type="link"
+			button-role="secondary"
+			target="_blank"
+			href="https://github.com/cds-snc/gcds-components"
+			lang="{{ locale }}"
+		>
+			{{ helpus[locale].github }}
+		</gcds-button>
+	</li>
+</ul>
```
Comment (melaniebmn) [inline src/_includes/partials/helpus.njk:33]: The spacing looks a bit wonky in this file, do you mind taking a look at this?

**Review by melaniebmn — APPROVED** [review]
LGTM, the spacing in that one file looks a bit off but otherwise feel free to merge!

File: src/_includes/partials/helpus.njk (hunk 5 above)
Comment (ethanWallace) [reply src/_includes/partials/helpus.njk]: I do not know why GitHub is displaying this way

**Pull request merged**